Northstar.CustomServers/mod/scripts/vscripts/sh_loadouts.nut

```
@@ -2812,6 +2812,9 @@ string function GetParentLoadoutProperty( string loadoutType, string propertyNam
 
 int function GetPersistentSpawnLoadoutIndex( entity player, string loadoutType )
 {
+	if (!IsValid ( player ) )

```

Like @Zanieon said, this should probably be moved to [sh_progression:172](https://github.com/R2Northstar/NorthstarMods/blob/e616cb1d5f148bbc1b892cac140a0138735ed05c/Northstar.CustomServers/mod/scripts/vscripts/sh_progression.nut#L172) to fix that specific issue, Though its arguably worth having a check here in general since something else could call it wrong (though I'd rather it fail if we are passing invalid entities here, there is no reason we should be)

---

Moving the check to sh_progression doesn't seem to fix it on my end. I'm probably doing something wrong

https://github.com/user-attachments/assets/594f4617-16b2-4449-bf54-396b31af997a

---

Nevermind, i was just being stupid. It works now

---

Why is this file still being modified? The problem does not lie in here.